### PR TITLE
[LP-2290] Fix all linting errors for student_account

### PR DESCRIPTION
**Pull request by muak50**
### Description

[LP-2290](https://philanthropyu.atlassian.net/browse/LP-2290)

### CODE REVIEW

File: openedx/features/student_account/forms.py [hunk 1]
```diff
@@ -72,6 +72,7 @@ def clean_opt_in(self):
         return cleaned_opt_in
 
     def clean_organization_name(self):
+        """Validates organization name"""
```
Comment (zahid-arbisoft) [inline openedx/features/student_account/forms.py:75]: Let use [edx recommended style](https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#docstrings)

```
"""
Validates organization name
"""
```

File: openedx/features/student_account/forms.py [hunk 2]
```diff
@@ -2,8 +2,8 @@
 Utility functions for validating custom form
 """
 import json
-import requests
 
+import requests
```
Comment (MunamMubashir-arbisoft) [inline openedx/features/student_account/forms.py:6]: import grouping needs to be fixed.

Comment (MunamMubashir-arbisoft) [inline openedx/features/student_account/forms.py:6]: Imports grouping needs to be fixed.

File: openedx/features/student_account/helpers.py [hunk 3]
```diff
@@ -1,31 +1,40 @@
...
+from student.models import CourseEnrollment
 
+from .constants import NON_ACTIVE_COURSE_NOTIFICATION, TOP_REGISTRATION_COUNTRIES
 
 log = logging.getLogger("edx.student")
 
 
 def get_non_active_course(user):
+    """
+    Get non active course fro given user
```
Comment (zahid-arbisoft) [inline openedx/features/student_account/helpers.py:30]: Lets fix typo `fro`

File: openedx/features/student_account/helpers.py [hunk 4]
```diff
@@ -38,7 +47,8 @@ def get_non_active_course(user):
         today = datetime.now(utc).date()
 
         try:
-            course = CourseOverview.objects.get(id=user_course.course_id, end__gte=today)
+            # course = CourseOverview.objects.get(id=user_course.course_id, end__gte=today)
```
Comment (zahid-arbisoft) [inline openedx/features/student_account/helpers.py:50]: Lets remove commented code

File: openedx/features/student_account/helpers.py [hunk 5]
```diff
@@ -130,13 +158,24 @@ def set_opt_in_and_affiliate_user_organization(user, form):
...
 
 
 def compose_and_send_activation_email_custom(request, registration, user):
+    """
+    Compose and send account activation email.
+
+    Arguments:
+        request (Request): Request Object
+        user (User): Django User object
+        registration (Registration): Registration object for the currently logged-in user
```
Comment (zahid-arbisoft) [inline openedx/features/student_account/helpers.py:174]: Please match the order of params with params in docstring

File: openedx/features/student_account/helpers.py [hunk 6]
```diff
@@ -1,31 +1,40 @@
...
+"""
 import logging
 import operator
-
 from datetime import datetime
+
+from django.db.utils import IntegrityError
 from pytz import utc
 
 import third_party_auth
```
Comment (MunamMubashir-arbisoft) [inline openedx/features/student_account/helpers.py:11]: Imports need to be fixed.

File: openedx/features/student_account/helpers.py [hunk 7]
```diff
@@ -38,7 +47,8 @@ def get_non_active_course(user):
         today = datetime.now(utc).date()
 
         try:
-            course = CourseOverview.objects.get(id=user_course.course_id, end__gte=today)
+            # course = CourseOverview.objects.get(id=user_course.course_id, end__gte=today)
+            course = CourseOverview.objects.get(id=user_course.course_id)
```
Comment (MunamMubashir-arbisoft) [inline openedx/features/student_account/helpers.py:51]: Would it work the same without adding the `end__gte` check?

File: openedx/features/student_account/helpers.py [hunk 8]
```diff
@@ -71,6 +81,15 @@ def save_user_utm_info(req, user):
     :return:
     """
     def extract_param_value(request, param_name):
+        """
+        Get value of params from request
+
+        Arguments:
+            request (Request): Request Object
```
Comment (zahid-arbisoft) [inline openedx/features/student_account/helpers.py:88]: Please add missing para `param_name` in dosctring

**Note: Please run pylint again, this and similar issues will be highlighted in pylint report**

File: openedx/features/student_account/helpers.py (hunk 3 above)
Comment (MunamMubashir-arbisoft) [inline openedx/features/student_account/helpers.py:30]: for*

File: openedx/features/student_account/tests/test_display_notification.py [hunk 9]
```diff
@@ -1,18 +1,25 @@
...
 from lms.djangoapps.onboarding.helpers import get_current_utc_date
 from openedx.core.djangoapps.timed_notification.core import get_course_first_chapter_link
+from openedx.features.course_card.tests.helpers import initialize_test_user, set_course_dates
+from openedx.features.student_account.constants import NON_ACTIVE_COURSE_NOTIFICATION
 from openedx.features.student_account.helpers import get_non_active_course
 from student.tests.factories import CourseEnrollmentFactory
-from openedx.features.course_card.tests.helpers import initialize_test_user, set_course_dates
 from xmodule.modulestore import ModuleStoreEnum
 from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
 from xmodule.modulestore.tests.factories import CourseFactory
```
Comment (MunamMubashir-arbisoft) [inline openedx/features/student_account/tests/test_display_notification.py:15]: Imports need to be sorted.

**Review by MunamMubashir-arbisoft — CHANGES_REQUESTED** [review]
Kindly run isort on all the files.

File: openedx/features/student_account/views.py [hunk 10]
```diff
@@ -1,33 +0,0 @@
-""" Views for a student's account information. """
-from django.http import HttpRequest
-from django.core.urlresolvers import resolve
-
-
-def local_server_get(url, session):
```
Comment (zahid-arbisoft) [inline openedx/features/student_account/views.py:6]: Are you sure this function is not being used anywhere?

**Review by zahid-arbisoft — CHANGES_REQUESTED** [review]
@muak50 Please update the JIRA story no in the link in the PR description. Also please run pylint again to fix remaining issues

File: openedx/features/student_account/helpers.py (hunk 7 above)
Comment (MunamMubashir-arbisoft) [reply openedx/features/student_account/helpers.py:51]: Also, please remove the commented out code.

Comment (muak50) [reply openedx/features/student_account/helpers.py:51]: It was commented to test... That needs to be removed.

File: openedx/features/student_account/views.py (hunk 10 above)
Comment (muak50) [reply openedx/features/student_account/views.py:6]: Yes, we are

**Review by zahid-arbisoft — APPROVED** [review]

**Review by MunamMubashir-arbisoft — APPROVED** [review]

**Pull request merged**